README.mkd

```diff
@@ -308,7 +308,11 @@ let g:gitgutter_sign_allow_clobber = 1
 
 #### Signs' colours and symbols
 
-By default vim-gitgutter uses your colourscheme's `Diff*` highlight groups' foreground colours for the signs' foreground colours.  For example, your `DiffAdd` foreground colour will be used for the `+` sign's foreground colour.
+The following rules are used by vim-gitgutter to choose the signs' colours:
+
+1. If your colourscheme defines the highlight groups `GitGutterAdd`, `GitGutterChange`, `GitGutterDelete`, etc., they will be used.
```

Comment:
"If your vimrc or colorscheme..."

Reply:
No … as I mentioned in the discussion, defining the highlight groups in .vimrc does not work (for me).

Reply:
Re-testing shows defining the highlight groups in .vimrc works in Linux, but not in GUI MacVim (only). It might be a Mac-GUI-related bug.